feature-group-definitions/fetch.yml

```diff
@@ -0,0 +1,40 @@
+name: Fetch (initial support)
+description: Fetch makes HTTP requests.
```

Comment:
Do you want to use more words, and the syntax `fetch()`?

Reply:
Looks good! I predict that someone will point out that it's not limited to HTTP, but this is the 99% case, so I think it's worth saying it.